Automattic/wp-calypso
# Add Fresh Start 2021 sale banner

### Changes proposed in this Pull Request

This PR implements the Fresh Start 2021 sale banner.

Fixes 1196341175636977-as-1199527862646240

### Implementation notes

- Removed New Year 2021 sale banner
- Added Fresh Start 2021 sale banner instead (campaign run from Feb 1 through Feb 14)
- Banner has no translation at the moment
- No experiment has been implemented at the moment

### Testing instructions

- Download the PR
- Run both Calypso and cloud locally
- Open your console
- Check that you see the new banner (both on mobile and desktop) in the Calypso plans (`/plans/:site`) and connect page (`/jetpack/connect/store`), and in the cloud pricing page (`/pricing`)
- Select another language than English in your profile
- Check that you still see the banner, in English
- Add the query parameter and value `?freshstart=false`, and verify that you don't see the banner
- Verify that there's no error in the console

### Screenshots

_Cloud mobile_
<img width="375" alt="Screen Shot 2021-02-01 at 3 23 53 PM" src="https://user-images.githubusercontent.com/1620183/106514864-d373e700-64a2-11eb-97c4-7e90433066e2.png">

_Cloud desktop_
<img width="1117" alt="Screen Shot 2021-02-01 at 3 23 44 PM" src="https://user-images.githubusercontent.com/1620183/106514884-da025e80-64a2-11eb-87bd-e877a2b15efb.png">
